### PR TITLE
Automation wizard

### PR DESCRIPTION
**Pull request by jeswanke**

### CODE REVIEW

File: frontend/src/routes/Applications/CreateApplication/CreateApplicationArgo.tsx [hunk 1]
```diff
@@ -41,9 +41,9 @@ export function WizardSyncEditor() {
     const { update } = useData() // Wizard framework sets this context
     return (
         <SyncEditor
+            editorTitle={'Argo YAML'}
```
Comment (KevinFCormier) [inline frontend/src/routes/Applications/CreateApplication/CreateApplicationArgo.tsx:44]: @fxiang1 You mentioned something about certain Argo terms not being used in the UI. Should this say **Application set YAML**

Comment (fxiang1) [reply frontend/src/routes/Applications/CreateApplication/CreateApplicationArgo.tsx:44]: Yep doc has been pushing to move away from anything referencing ArgoCD (see https://github.com/stolostron/backlog/issues/20621) so I think we should rename it to `Application set YAML` or something similar.

File: frontend/src/routes/Governance/policy-sets/EditPolicySet.tsx [hunk 2]
```diff
@@ -29,9 +29,9 @@ export function WizardSyncEditor() {
     const { update } = useData() // Wizard framework sets this context
     return (
         <SyncEditor
+            editorTitle={'Policy Set YAML'}
```
Comment (KevinFCormier) [inline frontend/src/routes/Governance/policy-sets/EditPolicySet.tsx:32]: I think it should be a lowercase 's' in 'Set'

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks John!

**Pull request merged**